src/state/common.rs

```diff
@@ -137,16 +137,13 @@ impl<P: MultiObjectiveProblem> Default for ParetoFront<P> {
     }
 }
 
-#[derive(Deref, DerefMut, Tid)]
-pub struct Loop(pub bool);
-impl CustomState<'_> for Loop {}
-
 #[derive(Default, Tid)]
-pub struct Population<P: Problem + 'static> {
+pub struct Populations<P: Problem + 'static> {
     stack: Vec<Vec<Individual<P>>>,
 }
```

Comment:
Needs documentation. (maybe separate issue)

Reply:
Yeah, we really need more documentation on almost everything. I think this counts as part of #96.